lib/rspec/mocks/configuration.rb

```diff
@@ -153,6 +154,11 @@ def verify_partial_doubles?
         @verify_partial_doubles
       end
 
+      # @private
+      # Used to track wether we are temporarily suppressing verifying partial
+      # doubles with `without_verifying_partial_doubles { ... }`
+      attr_accessor :temporarily_suppressing_partial_double_verification
```

Comment:
I think it's a bit odd to have the `-ing` form of the verb in a boolean flag like this.  How about we shorten this to `temporarily_suppress_partial_double_verification`?